crates/bevy_animation/src/lib.rs

```diff
@@ -980,13 +980,21 @@ fn trigger_untargeted_animation_events(
     }
 }
 
+/// Marker component for animations that should not use the default animation time.
+/// For more details, see the [`specify_animation_system`] documentation.
+#[derive(Debug, Component, Reflect, Clone, Copy, Default)]
+#[reflect(Component)]
+pub struct DontUseDefaultAnimationTime;
```

Comment:
Bikeshedding help please: I don't like this name very much. It's extremely long, and the missed apostrophe bothers me.